README.md

```diff
@@ -70,6 +63,13 @@ Libraries which enable you to provision specific resources. They are responsible
 * File an issue via [Github Issues](https://github.com/Azure/azure-sdk-for-java/issues/new/choose).
 * Check [previous questions](https://stackoverflow.com/questions/tagged/azure-java-sdk) or ask new ones on StackOverflow using `azure-java-sdk` tag.
 
+## Release branches (Release tagging)
+
+### Master branch
+The master branch has the most recent code with new features and bug fixes. It does **not** represent latest released **GA** SDK. See [below](#Client-GA-November-2019-Releases) for latest **GA** release.<br/>
+For each package we release there will be a unique git tag created that contains the name and the version of the package to mark the commit of the code that produced the package. This tag will be used for servicing via hotfix branches as well as debugging the code for a particular preview or stable release version.
```

Comment:
1. Can you move "For each package we release" under the Release branches (Release tagging) section, 
2. make the Release branches and the Master branch headings the same heading level (probably ###)
and 3. consider swapping their order

When I gave the feedback on the previous PR that merged this, I imagined Master Branch and Release Branches would be sibling headers, not nested headers.

Reply:
@joshfree - fixed. Does this work?

Reply:
LGTM